package.json

```diff
@@ -3,24 +3,26 @@
   "description": "User application for the Digital Marketplace",
   "version": "1.0.0",
   "private": true,
-  "engine": "node >= 6.10.3 < 7.0.0",
+  "engines": {
+    "node": "~ 10.16.3"
```

Comment:
Can we just use `^` here instead? It's not the job of this file to do the actual installing is it?

Reply:
(in fact it would have to be `^ 10.15` - nix stable has a patched 10.15 version)